themes/V3/5ePHB/snippets.js

```diff
@@ -27,35 +27,166 @@ module.exports = [
+									}}\n`,
+
+							},
+							{
+								name : 'Include in ToC up to H4',
+								icon : 'fas fa-dice-four',
+								gen  : dedent `\n{{tocDepthH4
+									}}\n`,
+							},
+							{
+								name : 'Include in ToC up to H5',
+								icon : 'fas fa-dice-five',
+								gen  : dedent `\n{{tocDepthH5
+									}}\n`,
+							},
+							{
+								name : 'Include in ToC up to H6',
+								icon : 'fas fa-dice-six',
+								gen  : dedent `\n{{tocDepthH6
+									}}\n`,
+							},
+						]
 					},
 					{
-						name : 'Include in ToC up to H5',
-						icon : 'fas fa-dice-five',
-						gen  : dedent `\n{{tocDepthH5
+						name : 'Table of Contents Individual Exclusion',
+						icon : 'fas fa-book',
+						gen  : dedent `\n{{tocExcludeH1 \n
 							}}\n`,
+						subsnippets : [
+							{
+								name : 'Individual Exclusion H1',
+								icon : 'fas fa-book',
+								gen  : dedent `\n{{tocExcludeH1 \n
+									}}\n`,
+							},
+							{
+								name : 'Individual Exclusion H2',
+								icon : 'fas fa-book',
+								gen  : dedent `\n{{tocExcludeH2 \n
+									}}\n`,
+							},
+							{
+								name : 'Individual Exclusion H3',
+								icon : 'fas fa-book',
+								gen  : dedent `\n{{tocExcludeH3 \n
+									}}\n`,
+							},
+							{
+								name : 'Individual Exclusion H4',
+								icon : 'fas fa-book',
+								gen  : dedent `\n{{tocExcludeH4 \n
+									}}\n`,
+							},
+							{
+								name : 'Individual Exclusion H5',
+								icon : 'fas fa-book',
+								gen  : dedent `\n{{tocExcludeH5 \n
+									}}\n`,
+							},
+							{
+								name : 'Individual Exclusion H6',
+								icon : 'fas fa-book',
+								gen  : dedent `\n{{tocExcludeH6 \n
+									}}\n`,
+							},
+						]
 					},
+
 					{
-						name : 'Include in ToC up to H6',
-						icon : 'fas fa-dice-six',
-						gen  : dedent `\n{{tocDepthH6
-							}}\n`,
+						name : 'Table of Contents Toggles',
+						icon : 'fas fa-book',
+						gen  : dedent`/* Add ToC include for H3, H4, H5, and H6 level Headers */
+									.page h3, .page h4, .page h5, .page h6 {
+										--TOC: include;
+									}\n\n`,
+						subsnippets : [
+							{
+								name : 'Enable H1-H4 all pages',
+								icon : 'fas fa-dice-four',
+								gen  : dedent`/* Add ToC include for H3 and H4 level Headers */
+											.page h3, .page h4 {
+												--TOC: include;
+											}\n\n`
+							},
+							{
+								name : 'Enable H1-H5 all pages',
+								icon : 'fas fa-dice-five',
+								gen  : dedent`/* Add ToC include for H3, H4, and H5 level Headers */
+											.page h3, .page h4, .page h5 {
+												--TOC: include;
+											}\n\n`
+							},
+							{
+								name : 'Enable H1-H6 all pages',
+								icon : 'fas fa-dice-six',
+								gen  : dedent`/* Add ToC include for H3, H4, H5, and H6 level Headers */
+											.page h3, .page h4, .page h5, .page h6 {
+												--TOC: include;
+											}\n\n`
+							},
```

Comment:
We want to replace these with the `.tocGlobal` classes then, right? Or remove `.tocGlobal` classes and leave these? They are identical behavior.

Reply:
replace.  That was sloppy of me.